indexer/services/roundtable/src/tasks/subaccount-username-generator.ts

```diff
@@ -32,3 +35,16 @@ export default async function runTask(): Promise<void> {
     }
   }
 }
+
+function generateUsername(): string {
+  const randomAdjective: string = adjectives[randomInt(0, adjectives.length)];
+  const randomNoun: string = nouns[randomInt(0, nouns.length)];
+  const randomNumber: string = randomInt(0, 1000).toString().padStart(
+    config.SUBACCOUNT_USERNAME_NUM_RANDOM_DIGITS, '0');
+
+  const capitalizedAdjective: string = randomAdjective.charAt(
+    0).toUpperCase() + randomAdjective.slice(1);
+  const capitalizedNoun: string = randomNoun.charAt(0).toUpperCase() + randomNoun.slice(1);
+
+  return `${capitalizedAdjective}${capitalizedNoun}${randomNumber}`;
```

Comment:
Can you include a unit test that checks that the capitalization, number and length conventions are followed?